src/icons.scss

```diff
@@ -687,4 +687,27 @@
     @extend .fa;
     @extend .fa-refresh;
   }
+
+  // Misc
+
+  &-search {
+    @extend .fa;
+    @extend .fa-fw;
+    @extend .fa-search;
+  }
+
+  // Alias
+
+  &-storage {
+    @extend .xo-icon-menu-new-sr;
+  }
+  &-settings {
+    @extend .xo-icon-menu-settings;
+  }
+  &-summary {
+    @extend .xo-icon-new-vm-summary;
+  }
+  &-play {
+    @extend .xo-icon-vm-start;
```

Comment:
What I meant is that `play` is different from `vm-start` therefore one should not be defined on top of the other.